<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-MMP14QM" failures="1" tests="3" name="ErrorValidation" time="79.283" errors="0" timestamp="2024-08-30T17:53:12 IST">
  <testcase classname="Udemyacademy.PageObjects.ErrorValidations" name="loginErrorValidation" time="3.799"/>
  <testcase classname="Udemyacademy.PageObjects.ErrorValidations" name="productErrorValidation" time="12.283">
    <skipped/>
  </testcase> <!-- productErrorValidation -->
  <testcase classname="Udemyacademy.PageObjects.ErrorValidations" name="productErrorValidation" time="12.265">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Udemyacademy.PageObjects.ErrorValidations.productErrorValidation(ErrorValidations.java:61)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- productErrorValidation -->
</testsuite> <!-- ErrorValidation -->
